etna/heroes/forms.py

```diff
@@ -0,0 +1,36 @@
+from django.utils.html import format_html
+
+from wagtail.admin.forms import WagtailAdminPageForm
+
+class PageWithHeroMixinForm(WagtailAdminPageForm):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields['hero_image_decorative'].label = format_html("%s <p style='font-size: 11px;'>%s</p>" % (
```

Comment:
Thanks @jacobtoppm - @gtvj do you think we should incorporate this now, or add it as a task for later so that we have a process for adding TNA specific styles to the dashboard?

Reply:
Hi @AshTNA - I think it would be best to do it now. Looks quite straightforward (on the face of it).

Reply:
Thanks @gtvj - I've done this, I had to create a separate SASS file otherwise all of our toolkit styles would ruin the dashboard CSS. I've added this new file to the readme, and the deployment config file.